### PR TITLE
fix: profiles not published from the backpack

### LINKED ISSUE #461: [QA] Backpack not reflecting the changes made on the avatar
### **Explorer Alpha build version:**
**0.3.160-main-251ca6c**

### **Issue Description:**

By changing wearables on the backpack and closing, the changes are not being saved

### **Steps to reproduce:**

  1. Go to backpack
  2. Make some changes to the avatar's appearence 
  3. Close the backpack

### **Expected behaviour:**
Changes are reflected in world

### **Current behaviour:**
No changes visible

### **Reproduction rate:**
- [ ] < 10%
- [ ]  < = 25%
- [ ]  50%
- [ ]  < = 75%
- [X]  100%

### **Severity:**

- [ ] Cosmetic
- [ ] Minor
- [ ] Medium
- [ ] High
- [X] Critical
- [ ] Blocker

### **Priority:**
- [X] High
- [ ] Medium
- [ ] Low

### **Platforms / OS:**
- [ ] Windows 11
- [X] Windows 10
- [ ] Mac Sonoma
- [ ] Mac (specify the version on Additional Notes)

#### **Additional Notes:**

https://github.com/decentraland/unity-explorer/assets/139157886/d739c6fd-3948-4a1e-b43f-4df49ff3221d

### PR DESCRIPTION
**Pull request by lorux0**
## What does this PR change?

Fixes #461 #561 

There was an exception whenever the IPFS HashV1 function was performed:
```
NotSupportedException: System.Configuration.ConfigurationManager::GetSection
  at System.Configuration.ConfigurationManager.GetSection (System.String sectionName) [0x00000] in <00000000000000000000000000000000>:0 
  at Common.Logging.LogManager+<>c__DisplayClass31_0.<BuildLoggerFactoryAdapter>b__0 () [0x00000] in <00000000000000000000000000000000>:0 
  at Common.Logging.Configuration.ArgUtils+<>c__DisplayClass15_0.<Guard>b__0 () [0x00000] in <00000000000000000000000000000000>:0 
  at Common.Logging.Configuration.ArgUtils.Guard[T] (Common.Logging.Configuration.ArgUtils+Function`1[T] function, System.String messageFormat, System.Object[] args) [0x00000] in <00000000000000000000000000000000>:0 
  at Common.Logging.LogManager.BuildLoggerFactoryAdapter () [0x00000] in <00000000000000000000000000000000>:0 
  at Common.Logging.LogManager.get_Adapter () [0x00000] in <00000000000000000000000000000000>:0 
  at Common.Logging.LogManager.GetLogger[T] () [0x00000] in <00000000000000000000000000000000>:0 
  at Ipfs.MultiHash..cctor () [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Ipfs.HashingExtensions.IpfsHashV1 (System.Byte[] content) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Profiles.RealmProfileRepository.SetAsync (DCL.Profiles.Profile profile, System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Profiles.RealmProfileRepository.SetAsync (DCL.Profiles.Profile profile, System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Profiles.LogProfileRepository.SetAsync (DCL.Profiles.Profile profile, System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Profiles.LogProfileRepository.SetAsync (DCL.Profiles.Profile profile, System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Profiles.Self.SelfProfile.PublishAsync (System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Profiles.Self.SelfProfile.PublishAsync (System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
  at DCL.Backpack.BackpackEquipStatusController.<PublishProfile>g__PublishProfileAsync|12_0 (System.Threading.CancellationToken ct) [0x00000] in <00000000000000000000000000000000>:0 
```

IPFS.Core.dll uses `Common.Logging` and `Common.Logging.Core` dependencies as described here: https://www.nuget.org/packages/ipfs.core/#dependencies-body-tab
Looks like we are missing some configuration to get this working properly on the Windows build (not happening on editor).
The fix initializes `LogManager.Adapter` with a instance that ignores logs, so it avoids to load the adapter through the settings, which seems to be not supported.

## How to test the changes?

1. Follow issue steps

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/Profiles/Repository/RealmProfileRepository.cs [hunk 1]
```diff
@@ -60,6 +62,11 @@ public async UniTask SetAsync(Profile profile, CancellationToken ct)
                 await ipfs.PublishAsync(entity, ct, files);
                 profileCache.Set(profile.UserId, profile);
             }
+            catch (Exception e)
+            {
+                Debug.LogError("Could not save the profile");
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/Profiles/Repository/RealmProfileRepository.cs:67]: Please use `ReportHub`.

File: Explorer/Assets/DCL/Profiles/Self/SelfProfile.cs [hunk 2]
```diff
@@ -58,17 +59,22 @@ public async UniTask PublishAsync(CancellationToken ct)
             if (profile == null)
             {
                 profile = Profile.NewRandomProfile(web3IdentityCache.Identity?.Address.EnsureNotNull("Web Identity is not initialized"));
+                Debug.Log($"PublishProfile.null.random: {JsonUtility.ToJson(profile)}");
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/Profiles/Self/SelfProfile.cs:62]: `ReportHub` everywhere pls so we can disable logs by category

**Review by mikhail-dcl — CHANGES_REQUESTED** [review]

File: Explorer/Assets/DCL/Profiles/Repository/RealmProfileRepository.cs (hunk 1 above)
Comment (lorux0) [reply Explorer/Assets/DCL/Profiles/Repository/RealmProfileRepository.cs:67]: Dont worry, will revert this change, its a wip

**Review by anicalbano — APPROVED** [review]
🟢 Reviewed by QA

https://github.com/decentraland/unity-explorer/assets/139157886/3c360239-4da3-4d8a-982e-914d5edc9b7e


https://github.com/decentraland/unity-explorer/assets/139157886/e52fc950-69ee-4732-b566-e7d978273bd5

**Review by RominaMarchetti — APPROVED** [review]

File: Explorer/Assets/Plugins/common.logging.3.4.1/lib/net35.meta
Comment (mikhail-dcl) [inline Explorer/Assets/Plugins/common.logging.3.4.1/lib/net35.meta:1]: I think we should keep .NET Standard version only, you can remove everything else - it should not be referenced and used anyway

File: Explorer/Assets/Plugins/ipfs.core.0.55.0/lib/netstandard1.4/Ipfs.Core.dll.meta
Comment (mikhail-dcl) [inline Explorer/Assets/Plugins/ipfs.core.0.55.0/lib/netstandard1.4/Ipfs.Core.dll.meta:1]: Use everywhere the version as close as possible to .NET Standard 2.1, but not higher. Delete everything else

**Review by mikhail-dcl — DISMISSED** [review]

**Pull request merged**